src/main/java/ladder/view/ResultView.java

```diff
@@ -33,16 +33,12 @@ private void printPlayer(Players players) {
     }
 
     private void printLadder(Ladder ladder) {
-        IntStream.range(0, ladder.height())
-                .mapToObj(ladder::get)
-                .forEach(this::printLine);
+        ladder.getLines().forEach(this::printLine);
     }
 
-    private void printLine(Line line) {
+    private void printLine(LadderLine line) {
         System.out.print(BLANK_LINE);
-        IntStream.range(0, line.size())
-                .mapToObj(line::get)
-                .forEach(point -> System.out.print(COLUMN + getLine(point)));
+        line.getPoints().forEach(point -> System.out.print(COLUMN + getLine(point)));
```

Comment:
```suggestion
        line.getPoints()
            .forEach(point -> System.out.print(COLUMN + getLine(point)));
```
스트림은 기능단위로 줄바꿈하여 사용하는것을 권장드립니다 !